### PR TITLE
Exhibition header

### PR DESCRIPTION
**Pull request by jamesgorrie**
WIP

Sorry @davidpmccormick - created it so I can reference some of the code.

### CODE REVIEW

**Comment by davidpmccormick** [comment]
Be my guest. Also, if you fancy helping me get my Flow errors into a less painful state I’d be very grateful!

File: common/model/exhibitions.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ export type ExhibitionPromo = {|
   url: string,
   title: string,
   image: Image,
+  squareImage: ?Picture;
+  thinImage: ?Picture;
```
Comment (jamesgorrie) [inline common/model/exhibitions.js]: We've not added these to the [`GenericContentTypes`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/model/generic-content-fields.js).
As it's something we'll be using else where, could we add it there and to the [generic parsers](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/parsers.js#L506)?

File: common/views/components/BaseHeader/BaseHeader.js [hunk 2]
```diff
@@ -30,9 +36,11 @@ type Props = {|
 |}
 
 export function getFeaturedMedia(
-  fields: GenericContentFields
+  fields: GenericContentFields,
+  isPicture?: boolean
 ): ?FeaturedMedia {
-  const image = fields.promo && fields.promo.image;
+  const image = fields.promo && fields.promo.thinImage;
```
Comment (jamesgorrie) [inline common/views/components/BaseHeader/BaseHeader.js]: This is odd - as I genuinely cannot see where this is being set.

**Comment by davidpmccormick** [comment]
@wellcometrust/experience-dev 👀 pls

**Comment by jamesgorrie** [comment]
Superb! 👍

**Pull request merged**